lagom-descriptor-generator-sbt-plugin/src/sbt-test/plugin/multiclient-java/test

```diff
@@ -0,0 +1,12 @@
+# Structure of this test:
+# =======================
+
+# In this test a hand-made lagom service consumes many downstream services specified by many swagger-json files in
+# the hello-impl 'resources/' folder. Consumed services are used on a test.
+
+> clean
+> compile
+> test
+$ exists petstore-api/target/scala-2.11/openapi/main/java/com/example/PetApi.java
+$ exists petstore-api/target/scala-2.11/openapi/main/java/com/example/StatusEnum.java
+$ exists petstore-api/target/scala-2.11/openapi/main/java/com/example/Pet.java
```

Comment:
Previous comment is on the wrong snippet.
This `scripted` test uses the generator on an `xxx-impl` project to test the consumer end. It uses two different `swaggerXYZ.yaml` to generate two client Service descriptors.
It also serves to test parsing of YAML wagger format.